apps/cloud_inference/server_app/source/inference_hip_rocal.cpp

```diff
@@ -0,0 +1,1092 @@
+#include "inference.h"
+#include "netutil.h"
+#include "common.h"
+#include <thread>
+#include <chrono>
+#include <dlfcn.h>
+#include <sstream>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <opencv2/highgui.hpp>
+#include <numeric>
+
+#if USE_SSE_OPTIMIZATION
+#if _WIN32
+#include <intrin.h>
+#else
+#include <x86intrin.h>
+#include <immintrin.h>
+#endif
+#endif
+
+extern void VX_CALLBACK log_callback(vx_context context, vx_reference ref, vx_status status, const vx_char string[]);
+
+
+#if ENABLE_HIP
+InferenceEngineRocalHip::InferenceEngineRocalHip(int sock_, Arguments * args_, const std::string clientName_, InfComCommand * cmd, const std::string folderPath_)
+                  :InferenceEngineHip(sock_, args_, clientName_, cmd),
+                  hip_dev_prop{ nullptr }, hip_stream{ nullptr },
+                  queueDeviceInputMemIdle{ nullptr }, queueDeviceInputMemBusy{ nullptr },
+                  queueDeviceOutputMemIdle{ nullptr }, queueDeviceOutputMemBusy{ nullptr }
+{
```

Comment:
move this { to previous line. Please follow Google coding style for all new files. True for all similar functions